tmt/cli.py

```diff
@@ -1717,7 +1717,7 @@ def try_command(context: Context, image_and_how: str, **kwargs: Any) -> None:
 @pass_context
 @workdir_root_options
 @option(
-    '-i', '--id', metavar="ID",
+    '-i', '--id', metavar="ID", multiple=True,
```

Comment:
@skycastlelily can we update the help so it is clear it can be specified multiple times?